pyCGM_Single/Pipelines.py

```diff
@@ -121,11 +142,73 @@ def prep(trajs):
     return frames
 
 def clearMarker(data,name):
+    """Clear Markers function
+
+    Clears the markers in a given dictionary for a given name.
+
+    Parameters
+    ----------
+    data : dict 
+        Dictionaries of marker lists.
+            { [], [], [], ...}
+    name : str
+        Name for the specific marker to be cleared in data.
+    
+    Returns
+    -------
+    data : dict
+        Returns the original data dictionary with the cleared marker.
+    
+    Example
+    -------
+    >>> import numpy as np 
+    >>> from .Pipelines import clearMarker
+    >>> data = [{'LTIL': np.array([-268.1545105 ,  327.53512573,   30.17036057]), 
+    ... 'RFOP': np.array([ -38.4509964 , -148.6839447 ,   59.21961594])},
+    ... {'LTIL': np.array([-273.1545105 ,  324.53512573,   36.17036057]),
+    ... 'RFOP': np.array([ -38.4509964 , -148.6839447 ,   59.21961594])}]
+    >>> name = 'LTIL'
+    >>> clearMarker(data, name) #doctest: +NORMALIZE_WHITESPACE
+    [{'LTIL': array([nan, nan, nan]), 
+    'RFOP': array([ -38.4509964 , -148.6839447 ,   59.21961594])}, 
+    {'LTIL': array([nan, nan, nan]), 
+    'RFOP': array([ -38.4509964 , -148.6839447 ,   59.21961594])}]
+    """
     for i in range(len(data)):
         data[i][name] = np.array([np.nan,np.nan,np.nan])
     return data
     
 def filtering(Data):
+    """Filter function
+
+    Parameters
+    ----------
+    Data : dict 
+        Dictionaries of marker lists.
+            { [], [], [], ...}
+    
+    Returns
+    -------
+    data : dict
+    
+    Example
+    -------
+    >>> import numpy as np 
+    >>> from .Pipelines import filtering
+    >>> from .pyCGM_Helpers import getfilenames
+    >>> from .pycgmIO import loadData, dataAsDict
+    >>> motionData = loadData(getfilenames(x=2)[0])
+    Sample...
```

Comment:
Why is this outputing "Sample..."?

Reply:
Okay lets put that fill string in. As long as the string is relative to directory (its not your full path) then its fine to keep for now (although later it just shouldn't print anything).

Reply:
Okay, resolved.